src/models/stackedAreaChart.js

```diff
@@ -392,6 +394,15 @@ nv.models.stackedAreaChart = function() {
                         allData[indexToHighlight].highlight = true;
                 }
 
+                //If we are not in 'expand' mode, add a 'Total' row to the tooltip.
+                if (stacked.style() != 'expand' && allData.length >= 2) {
+                    allData.push({
+                        key: "TOTAL",
```

Comment:
There is no way to change the "TOTAL" string here?  What about for other languages?